pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -153,6 +153,20 @@ func (r *PyTorchJobReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		logger.Error(err, "Reconcile PyTorchJob HPA error")
 		return ctrl.Result{}, err
 	}
+	if pytorchjob.Spec.ElasticPolicy != nil {
+		elasticPolicy := pytorchjob.Spec.ElasticPolicy
+		schedulingPolicy := pytorchjob.Spec.RunPolicy.SchedulingPolicy
+		if schedulingPolicy != nil {
+			schedulingPolicy.MinAvailable = elasticPolicy.MinReplicas
```

Comment:
what if `schedulingPolicy.MinAvailable` is set while not equal to `elasticPolicy.MinReplicas`?